docroot/modules/humsci/hs_capx/tests/Unit/orgs.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "~0N`S$Z4",
```

Comment:
How'd you come up with this wonderful test data 😆

Reply:
It came to me in a dream :1st_place_medal: